lib/dataloader.ex

```diff
@@ -40,6 +45,38 @@ defmodule Dataloader do
   have a different source used for each context. This provides an easy way to
   enforce data access rules within each context. See the `DataLoader.Ecto`
   moduledocs for more details
+
+  ## Options
+
+  There are two configuration options:
+
+  * `timeout` - The maximum timeout to wait for running a source, defaults to
+    the 1s more than the maximum timeout of all added sources. Set with care,
+    timeouts should really only be set on sources.
+  * `get_policy` - This configures how the dataloader will behave when fetching
+    data which may have errored when we tried  to `load` it.
+
+  ### `get_policy`
+
+  There are three implemented behaviours for this:
+
+  * `raise_on_error` (default)- If successful, calling `get/4` or `get_many/4`
+    will return the value. If there was an exception when trying to load any of
+    the data, it will raise that exception
+  * `return_nil_on_error` - Behaves similar to `raise_on_error` but will just
+    return `nil` instead of `raising`. It will still log errors
```

Comment:
`It will still log error tuples`, extra `s` on error

Reply:
Nope! This is intentional, it does not log error tuples , it logs the error that actually happened. This is completely non-tupified behaviour, is there something I could do to make that clearer do you think?